README.md

```diff
@@ -19,3 +19,23 @@ The Tungsten Hackathon channeled the energy and excitement of DFINITY's internal
 [Shield](https://github.com/crusso/shield) - An app connecting at-risk people with nearby volunteers.
 
 [Hero](https://github.com/stanleygjones/hackathon) - A random "killer app" generator inspired by the Wu-Tang Name Generator.
+
+
+## Motoko Libraries
+
+### Data structures
+
+- [motoko-bigmap](https://github.com/dfinity/motoko-bigmap) - A map data structure that scales beyond the limits of single canisters and single networks.
+
+### Algorithms
+
+- [motoko-adapton](https://github.com/matthewhammer/motoko-adapton) - Dynamic dependence graph and memoization techniques in Motoko.
+- [motoko-redraw](https://github.com/matthewhammer/motoko-redraw) - 2D graphics and layout algorithms for Motoko.
+
+### Testing
+
+- [motoko-bigtest](https://github.com/dfinity/motoko-bigmap/tree/master/src/BigTest) - A generic testing DSL (written in Motoko) for Internet Computer services; currently part of [BigMap](https://github.com/dfinity/motoko-bigmap).
```

Comment:
Is this one public yet?

Reply:
Not yet.  I heard today that it may be as easy as we'd hoped to open things.

What steps are necessary now, given what we know from the `base` experience?